src/runtime/scheduler_tinygoriscv.S

```diff
@@ -1,32 +1,42 @@
+#ifdef RV64
```

Comment:
Aren't there already preprocessor macros for this? For example, it appears there is `__riscv_mul` (for the M extension) and `__riscv_xlen` which defines XLEN.

Reply:
I didn't know this, thank you.